packages/mask/src/plugins/Game/SNSAdaptor/GameShareDialog.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from 'react'
+import { makeStyles } from '@masknet/theme'
+import { Button, Typography, Box } from '@mui/material'
+import { activatedSocialNetworkUI } from '../../../social-network'
+import { useI18N } from '../../../utils'
+import type { GameInfo } from '../types'
+import { Share_Twitter } from '../constants'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        margin: theme.spacing(0, 2.5),
+        zIndex: 99999,
+    },
+    shareNotice: {
+        color: theme.palette.maskColor.main,
+        fontSize: '16px',
+        fontFamily: 'TwitterChirp',
+        lineHeight: '16px',
+        marginTop: theme.spacing(2),
+    },
+    shareButton: {
+        width: '100%',
+        margin: theme.spacing(4, 0, 3),
+    },
+}))
+
+interface PetSetDialogProps {
+    onClose: () => void
+    gameInfo: GameInfo | undefined
+}
+
+export default function GameShareDialog({ onClose, gameInfo }: PetSetDialogProps) {
+    const { t } = useI18N()
+    const { classes } = useStyles()
+
+    const shareText = `I'm playing ${gameInfo?.name} by @NonFFriend ${gameInfo?.twitterId} on my Twitter profile. Install the Mask Network Extension mask.io and JOIN ME!\n #mask_io #NFF #NFTgame\n${Share_Twitter}`
```

Comment:
Config it in `Game/locale/en-US.json`

```json
{
  "share_text": "I'm playing {{name}} by @NonFFriend {{twitter_id}} on my Twitter profile. Install the Mask Network Extension mask.io and JOIN ME!\n $t(mask:hash_tag) #NFF #NFTgame\n{{share_Twitter}}"
}
```


`$t(mask:hash_tag)` is nesting feature of i18next